notebooks/cuml_benchmarks.ipynb

```diff
@@ -0,0 +1,997 @@
+      "NearestNeighbors Speedup (n_samples=8192, n_features=64) = 89.59011354629887\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/share/software/miniconda3/envs/cuml_dev_014_2/lib/python3.7/site-packages/cuml-0.14.0a0+852.gc5770a0.dirty-py3.7-linux-x86_64.egg/cuml/utils/input_utils.py:188: UserWarning: Expected column ('F') major order, but got the opposite. Converting data, this will result in additional memory utilization.\n",
+      "  warnings.warn(\"Expected \" + order_to_str(order) + \" major order, \"\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "NearestNeighbors Speedup (n_samples=8192, n_features=128) = 70.65879409226257\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/share/software/miniconda3/envs/cuml_dev_014_2/lib/python3.7/site-packages/cuml-0.14.0a0+852.gc5770a0.dirty-py3.7-linux-x86_64.egg/cuml/utils/input_utils.py:188: UserWarning: Expected column ('F') major order, but got the opposite. Converting data, this will result in additional memory utilization.\n",
+      "  warnings.warn(\"Expected \" + order_to_str(order) + \" major order, \"\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "NearestNeighbors Speedup (n_samples=8192, n_features=256) = 106.57735670533263\n"
+     ]
+    }
+   ],
+   "source": [
+    "runner = cuml.benchmark.runners.SpeedupComparisonRunner(\n",
+    "    bench_rows=[2**x for x in range(11, 14)], \n",
+    "    bench_dims=[64, 128, 256],\n",
+    "    dataset_name=DATA_NEIGHBORHOODS,\n",
+    "    input_type=INPUT_TYPE,\n",
+    "    n_reps=N_REPS\n",
+    ")\n",
+    "\n",
+    "execute_benchmark(\"NearestNeighbors\", runner)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Nearest Neighbors"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### KNeighborsClassifier"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "runner = cuml.benchmark.runners.SpeedupComparisonRunner(\n",
+    "    bench_rows=[2**x for x in range(11, 24)], \n",
+    "    bench_dims=[64, 128, 256],\n",
+    "    dataset_name=DATA_CLASSIFICATION,\n",
+    "    input_type=INPUT_TYPE,\n",
+    "    n_reps=N_REPS\n",
+    ")\n",
+    "\n",
+    "execute_benchmark(\"KNeighborsClassifier\", runner)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### KNeighborsRegressor"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "algorithm = \"KNeighborsRegressor\"\n",
+    "runner = cuml.benchmark.runners.SpeedupComparisonRunner(\n",
+    "    bench_rows=[2**x for x in range(11, 24)], \n",
+    "    bench_dims=[64, 128, 256],\n",
```

Comment:
What about switching bench dims to something like `[32, 256]` - just using two sizes may be pretty informative (though faster) and increasing the range could help capture the skinny vs. wide difference?